### PR TITLE
TEST_0001 automated a search box

### PR DESCRIPTION
**Pull request by praneethsingireddy**
Adding a file

### CODE REVIEW

**Review by mahendra15nov — CHANGES_REQUESTED** [review]
pls add some comments

**Comment by udaytejeshwar** [comment]
good going

**Review by udaytejeshwar — CHANGES_REQUESTED** [review]
there is nothing to review here

File: djdknldm [hunk 1]
```diff
@@ -0,0 +1 @@
+hello mahendar anna
```
Comment (mahendra15nov) [inline djdknldm:1]: hfhdfh

**Review by mahendra15nov — CHANGES_REQUESTED** [review]

**Review by mahendra15nov — APPROVED** [review]

**Pull request merged**